### DISCUSSION TITLE
How does archiving work?

### DISCUSSION THREAD

**Discussion by Irene-UXXI** [flowsforapex/apex-flowsforapex#776] [Q&A] [answered]
Hi,

We have enabled the archiving option in the product. According to the documentation:

"The archive document is created by an archive process, which should be configured to run as an APEX Automation, typically on a daily (nightly) basis. The archive document is created once the process instance has completed, or reached terminated status. The archive summaries create one JSON document per process instance. The archived JSON file can be stored in either an Oracle table or in OCI Object Storage."

But we don't see anything being stored in the table we have configured. Is there a JOB or other configuration that needs to be enabled?

Thanks.

**Comment by nielsdebr** [comment, marked as answer]
Did you switch on the automation?

<img width="978" alt="Screenshot 2024-12-02 at 13 19 53" src="https://github.com/user-attachments/assets/ebbe5a81-05a5-4422-85fe-bd04b6e79d46">

**Reply by Irene-UXXI** [reply]
Thank you very much. Indeed, I was not active in the automation. Now it works ;-)

**Reply by rallen2010** [reply]
See also Issue #707 - which would have prevented you from configuring Statistics...
